GDJS/package.json

```diff
@@ -24,12 +36,13 @@
     "typescript": "4.3.2"
   },
   "scripts": {
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && cd ../Extensions/TileMap/helper && npm i",
+    "type-gen": "node scripts/generate-esm-types.js && prettier --write \"Runtime/types/auto-generated/*.d.ts\"",
```

Comment:
Uber nit: "generate-types" for consistency with the rest and follow the semi convention of using verbs for nom scripts. :)